aio/content/marketing/contributors.json

```
@@ -677,6 +677,14 @@
     "bio": "Nir is a Principal Frontend Consultant & Head of the Angular department at 500Tech, Google Developer Expert and community leader. He organizes the largest Angular meetup group in Israel (Angular-IL), talks and teaches about front-end technologies around the world. He is also the author of two books about Angular and the founder of the 'Frontend Band'.",
     "groups": ["GDE"]
   },
+  "nishugoel": {
+    "name": "Nishu Goel",
+    "picture": "nishugoel.jpg",
+    "twitter": "TheNishuGoel",
+    "website": "https://www.unravelweb.dev",
```

This URL doesn't seem to work. Did you mean:

```suggestion
    "website": "https://unravelweb.dev",
```